### PR TITLE
Unfork celery and update requirements

### PR DESCRIPTION
**Pull request by jvperrin**
Also update the pre-commit config to the 1.0+ version format.

This has a lot of version updates, so hopefully everything goes smoothly. I did run it and everything started fine and it connected to celery, but I didn't test it any further beyond that.

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+repos:
 -   repo: https://github.com/pre-commit/pre-commit-hooks.git
     sha: v0.7.1
     hooks:
```
Comment (chriskuehl) [inline .pre-commit-config.yaml:4]: you can drop all the language_version in this file now that we're on a new-enough pre-commit

File: create/tasks.py [hunk 2]
```diff
@@ -30,7 +30,7 @@
 celery.conf.broker_use_ssl = {
     'ssl_cert_reqs': ssl.CERT_NONE,
 }
-# `redis_backend_use_ssl` is an OCF patch which was proposed upstream:
+# `redis_backend_use_ssl` is an OCF patch which was merged upstream:
```
Comment (chriskuehl) [inline create/tasks.py]: probably don't need the comment now that it's part of upstream and documented in the regular docs?

**Review by chriskuehl — APPROVED** [review]

File: .pre-commit-config.yaml (hunk 1 above)
Comment (jvperrin) [reply .pre-commit-config.yaml:4]: Neat

File: create/tasks.py (hunk 2 above)
Comment (jvperrin) [reply create/tasks.py]: Yeah, I wondered about that too, I'll just remove it then.

**Review by chriskuehl — APPROVED** [review]

**Pull request merged**